podcasts/ShowNotesUpdater.swift

```diff
@@ -4,10 +4,14 @@ import PocketCastsUtils
 
 class ShowNotesUpdater {
     class func updateShowNotesInBackground(podcastUuid: String, episodeUuid: String) {
-        if FeatureFlag.newShowNotesEndpoint.enabled {
+        if FeatureFlag.newShowNotesEndpoint.enabled || FeatureFlag.transcripts.enabled {
             Task {
                 // Load the show notes and any available chapters
                 _ = try? await ShowInfoCoordinator.shared.loadChapters(podcastUuid: podcastUuid, episodeUuid: episodeUuid)
+
+                if FeatureFlag.transcripts.enabled {
```

Comment:
I know that the above will also load the transcripts metadata but just want be sure that this is called here even if the above is removed.